.github/workflows/tests.yml

```diff
@@ -102,7 +102,7 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Setup Node
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
         node-version: '16'
```

Comment:
> In scope of this release we changed version of node runtime for action from node16 to node20 and updated dependencies in [actions/setup-node#866](https://redirect.github.com/actions/setup-node/pull/866)

We hatdclode it here, but I _guess_ it would break with Node20, would not it? Or should we just try in a new PR?

Reply:
Okay, i just tried it in https://github.com/PrivateBin/PrivateBin/pull/1190